app_flutter/lib/widgets/task_icon.dart

```diff
@@ -32,35 +32,42 @@ class TaskIcon extends StatelessWidget {
     // different assets.
     final Color blendFilter = brightness == Brightness.dark ? Colors.white : null;
 
-    if (qualifiedTask.stage == StageName.luci && qualifiedTask.task == 'linux_bot') {
-      return Image.asset(
-        'assets/fuchsia.png',
-      );
-    }
-    switch (qualifiedTask.stage) {
-      case StageName.cirrus:
+    if (qualifiedTask.builder != null) {
+      if (qualifiedTask.builder.toLowerCase().contains('_fuchsia')) {
         return Image.asset(
-          'assets/cirrus.png',
+          'assets/fuchsia.png',
           color: blendFilter,
         );
-      case StageName.luci:
-        return Image.asset(
-          'assets/chromium.png',
```

Comment:
The web tests are missing. `contains('_web')` should be sufficient

Reply:
done